bbb/services.py

```diff
@@ -519,6 +519,10 @@ def handlePending(self, data, msg):
         # as running. The BuildbotListener will take care of that when a slave
         # actually picks up the job.
         else:
+            redelivered = msg.get("delivery_info", {}).get('redelivered')
+            if redelivered:
+                log.warning("Task %s was redelivered. delivery_info: %s",
+                            taskid, msg.get("delivery_info"))
```

Comment:
isn't this:
```Pulse msg for Task %s was redelivered``` based on the convo we had.  Since the redelivered arg comes from amqp itself and not TC. And TC also has code that allows a resend of the same message anyway.